torchvision/utils.py

```diff
@@ -575,3 +549,63 @@ def _make_ntuple(x: Any, n: int) -> Tuple[Any, ...]:
     if isinstance(x, collections.abc.Iterable):
         return tuple(x)
     return tuple(repeat(x, n))
+
+
+def _parse_colors(
+    colors: Optional[Union[List[Union[str, Tuple[int, int, int]]], str, Tuple[int, int, int], None]],
+    num_objects: int,
```

Comment:
Nit: Let's enforce keyword only here

```suggestion
    *,
    num_objects: int,
```